src/components/ConversationScreen.tsx

```
@@ -75,51 +101,341 @@ const ConversationScreen = () => {
+      }, 100);
+      
+    } catch (error) {
+      console.error('메시지 전송 실패:', error);
+      Alert.alert('오류', '메시지 전송에 실패했습니다.');
+    } finally {
+      setIsSending(false);
+    }
+  };
+
+  // 일기 생성
+  const createDiary = useCallback(async () => {
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)